games-arcade/caveexpress/caveexpress-2.4.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="Classic 2D platformer with physics-based gameplay and dozens of levels"
+DESCRIPTION="Master your pedal-powered flying machine to pick up packages \
+from your cave-dwelling clients and drop them off at the collection point. \
+But beware! Mighty mastodons, terrifying pterodactyls and others \
+would rather see you extinct."
+HOMEPAGE="https://caveproductions.org/"
+COPYRIGHT="2014-2016 Martin Gerhardy"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="3b15239c8e748ebf222562da5571fa9bb2fbeec4"
+SOURCE_URI="https://github.com/mgerhardy/caveexpress/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="5b6ed23830e13dba10cb4ae3edfed7a2ee728034b8c5e71fd37eb4abc5a3ab1d"
+SOURCE_DIR="caveexpress-$srcGitRev"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	caveexpress$secondaryArchSuffix = $portVersion
+	cavepacker$secondaryArchSuffix = $portVersion
+	app:caveexpress = $portVersion
+	app:cavepacker = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libatomic$secondaryArchSuffix
+	lib:libBox2D$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_image_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix
+	lib:libSDL2_net_2.0$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libyajl$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libBox2D$secondaryArchSuffix
+	devel:liblua$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libSDL2_image_2.0$secondaryArchSuffix
+	devel:libSDL2_mixer_2.0$secondaryArchSuffix
+	devel:libSDL2_net_2.0$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libyajl$secondaryArchSuffix
+	glm${secondaryArchSuffix}_devel
```

Comment:
gml isn't detected (at least not here), some other checks fail also (libraries that are searched for), but these don't break the build